tools/gbdk_infinity.diff

```diff
@@ -1,6 +1,6 @@
-diff -ru SDK.orig/gbz80-gb/2-1-5/lib/crt0.s SDK/gbz80-gb/2-1-5/lib/crt0.s
---- SDK.orig/gbz80-gb/2-1-5/lib/crt0.s	1999-10-17 22:02:46.000000000 -0700
-+++ SDK/gbz80-gb/2-1-5/lib/crt0.s	2016-07-25 14:14:31.603316904 -0700
+diff -ru sdk.orig/gbz80-gb/2.1.5/lib/crt0.s sdk/gbz80-gb/2.1.5/lib/crt0.s
+--- sdk.orig/gbz80-gb/2.1.5/lib/crt0.s	1999-10-17 22:02:46.000000000 -0700
++++ sdk/gbz80-gb/2.1.5/lib/crt0.s	2016-07-25 14:14:31.603316904 -0700
```

Comment:
I had the same issue when I first checked out the project and didn't realize this was the issue at first though, so there is some merit to this change: https://github.com/infinity-gbc/infinity/issues/3